### ISSUE TITLE
Image Orientation

### ISSUE THREAD

**Issue by lulalala** [lulalala/mei#17]
Some images, while viewing on its own, has the correct orientation, but when it is displayed in html web page, the orientation is wrong.

**Comment by szTheory** [comment]
@lulalala can you provide a sample image that causes this problem?

**Comment by lulalala** [comment]
Here is an image: http://board.lulalala.com/uploads/36883/1523093867_apple.jpeg

And I finally found an answer to this:

https://stackoverflow.com/questions/42401203/chrome-image-exif-orienation-issue

So basically all browsers will only respect EXIF orientation if it is displayed by itself (instead of inside an html file). This issue has been there since [2010](https://bugs.chromium.org/p/chromium/issues/detail?id=56845).

So basically Mei has to process image so it is in the correct orientation without looking up to EXIF info.
